be/src/formats/parquet/group_reader.cpp

```diff
@@ -274,6 +275,14 @@ Status GroupReader::_create_column_reader(const GroupReaderParam::Column& column
             RETURN_IF_ERROR(ColumnReader::create(_column_reader_opts, schema_node, column.slot_type(),
                                                  column.t_iceberg_schema_field, &column_reader));
         }
+        if (column_reader == nullptr) {
```

Comment:
Can you write a ut to cover new lines here?
I'm just curious about what the situation can reach here.

You can refer ut to `iceberg_schema_evolution_file_reader_test.cpp`